resolwe_bio/tools/create_track_hubs.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+# pylint: disable=missing-docstring,invalid-name
+"""Create UCSC track hub files."""
+
+import argparse
+import os
+
+
+def get_args():
+    """Parse command-line arguments."""
+    parser = argparse.ArgumentParser(description="Create files for a UCSC track hub")
+    parser.add_argument('--name', nargs='+', help="Sample names")
+    parser.add_argument('--track_hub_name', help="Track hub name")
+    parser.add_argument('--build', help="Genome build")
+    parser.add_argument('--browser_url', help="Local UCSC genome browser URL")
+    parser.add_argument('--email', help="User email")
+    return parser.parse_args()
+
+
+def main():
+    """Create track hub files."""
+    args = get_args()
+    build = args.build
+    name = args.track_hub_name
+    email = args.email
+    if not email:
+        email = 'support@genialis.com'
+
+    trackdb_file = os.path.join(build, 'trackDb.txt')
+    genome_file = '.'.join([name, 'genomes', 'txt'])
+    with open(genome_file, 'w') as outfile:
+        outfile.write('genome {}\ntrackDB {}'.format(build, trackdb_file))
+
+    hub_file = '{}.hub.txt'.format(name)
+    with open(hub_file, 'w') as outfile:
+        for label in ['hub', 'shortLabel', 'longLabel']:
+            outfile.write(' '.join([label, name]) + '\n')
+        sequence = '\n'.join([' '.join(['genomeFile', genome_file]), ' '.join(['email', email])])
+        outfile.write(sequence)
+
+    URL = 'http://taco-wiki.grid.bcm.edu/genialis/ucsc'
```

Comment:
This cannot hardcoded into a open-source process.

Reply:
How do you suggest we solve this without user having to know this information? @mstajdohar any suggetions?